kuma/contributions/tasks.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from celery.task import task
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from django.utils import translation
+
+from kuma.core.decorators import skip_in_maintenance_mode
+from kuma.core.email_utils import render_email
+
+
+log = logging.getLogger('kuma.contributions.tasks')
+
+
+@task
+@skip_in_maintenance_mode
+def contribute_thank_you_email(username, user_email):
+    """Create a notification email for new contributor."""
+    message_context = {
+        'user_email': user_email,
+        'username': username,
+        'site': settings.SITE_URL
+    }
+
+    locale = settings.WIKI_DEFAULT_LANGUAGE
```

Comment:
This is OK for now, we'll need to change it when we ship translations in a few weeks.